lib/router.js

```diff
@@ -301,6 +306,10 @@ Router.map( function () {
     $( ".helpitem" ).removeClass( "helpNavSelected" );
     $( ".helpitem" ).eq(4).addClass( "helpNavSelected" );
     this.next();
+
+    if ( ! Meteor.userId() ) {
+        this.render
```

Comment:
Expected an assignment or function call and instead saw an expression.<br>Missing semicolon.